autoscale/src/main/java/com/sequenceiq/periscope/service/AltusMachineUserService.java

```diff
@@ -97,4 +101,9 @@ private void syncEnvironment(String accountId, String machineUserCrn, String env
         LOGGER.info("Finished invoking freeIpa user Sync with operation: {}, for environmentCrn: {} and machineUserCrn: {}", statusResponse.getOperationId(),
                 environmentCrn, machineUserCrn);
     }
+
+    private String encodeEnvironmentCrnForMachineUser(String environmentCrn) {
```

Comment:
Is there a utility class of some kind in a module shared by autoscale and core where this functionality can go ?

Reply:
Moved this to a separate class in the `common` module